rasa/shared/core/events.py

```diff
@@ -111,18 +111,6 @@ def format_message(
     )
 
 
-def first_key(d: Dict[Text, Any], default_key: Any) -> Any:
```

Comment:
I guess you've checked that everything that is in `rasa.shared.*` and removed in this PR isn't used by Rasa X?

Reply:
yes!